src/app.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-cycle */
+import { landingPage } from './pages/landing-page.js';
+import { signUpPage } from './pages/sign-up.js';
+import { signInPage } from './pages/sign-in.js';
+import { addInfoPage } from './pages/add-info.js';
+import { feed } from './pages/feed.js';
+
+const routes = {
+  '/': landingPage,
+  '/sign-up': signUpPage,
+  '/sign-in': signInPage,
+  '/add-info': addInfoPage,
+  '/home': feed,
```

Comment:
```suggestion
  '/': Landing,
  '/sign-up': SignUp,
  '/sign-in': SignIn,
  '/add-info': AddInfo,
  '/home': Home,
```
Al ser un componente podrías nombrarlo con la primer letra en mayúscula.